dashboard/app/controllers/api/v1/assessments_controller.rb

```diff
@@ -24,7 +24,7 @@ def load_from_cache
   # GET '/dashboardapi/assessments'
   def index
     # Only authorized teachers have access to locked question and answer data.
-    render status: :forbidden unless current_user.authorized_teacher?
+    render status: :forbidden unless current_user && current_user.authorized_teacher?
```

Comment:
i recently learned about `current_user&.authorized_teacher?` syntax. do we prefer that for this type of check?

Reply:
yes! thanks!